.gitignore

```diff
@@ -2,6 +2,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+*.mjs
```

Comment:
I wouldn't ignore all `.mjs` files because at the end it's the format of an ES6 module. I think that something like `vite.config.ts.timestamp*` would be better

Reply:
Done.